app/models/vm.rb

```diff
@@ -1,6 +1,7 @@
 class Vm < VmOrTemplate
   default_scope { where(:template => false) }
-
+  has_one :container_deployment, :through => :container_deployment_node
+  has_one :container_deployment_node
```

Comment:
@Fryguy I don't think we ever settled on a solution for this...

The problem (as I understand it) is that Conatiners need to reflect the fact that some of their deployments are related to VMs in other providers.  @alongoldboim is that a fair assessment of this?

Adding these relationships at the base level means that any VM in any provider can have a `container_deployment`.  At one point we talked about having the concept of duplicating these "linked" `VM`s.  So, in this case, containers would have a `VM` object (namespaced under `ManageIQ::Providers::ConatinerManager`).  And, that container `VM` would have some link to another provider's `VM` object to say "these are basically the same thing".

@Fryguy I guess for now we have to stick with what @alongoldboim has here, unless you have another idea.